src/agenda/index.js

```diff
@@ -15,11 +15,7 @@ import {AGENDA_CALENDAR_KNOB} from '../testIDs';
 const HEADER_HEIGHT = 104;
 const KNOB_HEIGHT = 24;
 //Fallback for react-native-web or when RN version is < 0.44
-const {Text, View, Dimensions, Animated, ViewPropTypes} = ReactNative;
-const viewPropTypes =
-  typeof document !== 'undefined'
-    ? PropTypes.shape({style: PropTypes.object})
-    : ViewPropTypes || View.propTypes;
+const {Text, View, Dimensions, Animated, Platform} = ReactNative;
```

Comment:
Why is 'import * as ReactNative from 'react-native';' still necessary? Can't we extract the components from 'react-native' without importing it all?

Reply:
I was trying to be as non-invasive as possible - I'm new to JS / React. Would you like me to remove the 'import * as ReactNative' and fix anything that depends on it?

Reply:
I do want to remove the * import but I don't want to break anyone. Can you tell me how do you test this in web on old RN versions?

Reply:
Err.. I haven't tested on older RN versions. Good idea. I'll make sure that works then post back here.